### DISCUSSION TITLE
Trouble Importing an Interactive 3d model from R

### DISCUSSION THREAD

**Discussion by zzoligaszz** [community/community#53714] [New to GitHub]
Hi,

I have created an interactive 3d model of an image in R and exported it into html format using R markdown.

I need to embed this interactive 3d model into a scientific report I am writing in Word format.

My end goal is to embed this 3d model in the report, but maintain the interactive format.

I have researched various methods, and Github seems to be the answer but I am having serious trouble 1) getting the model to show anywhere when viewing the repository page. 404 error page displays. 2) I still have no idea how to embed this into the report.

Is Github the right method for this, and if so, please could somebody steer me in the right direction.

Thank you for any help you can provide.

**Comment by cyrusjameskhan** [comment]
I haven't tried it out myself because I don't have an upgraded account but you could give the following a try:

1. Upload your 3D model to the repository

2. Enable GitHub Pages for the repository

3. Embed your 3D model into your scientific report using an iframe.
